doc/changes_from_dnf4.7.rst

```diff
@@ -230,6 +230,7 @@ Changes to individual commands
 
 ``makecache``
   * Metadata is now stored in different directories, see the ``cachedir`` configuration option :ref:`changes <cachedir_option_conf_changes_ref-label>` for more details.
+  * The ``--timer`` option has been dropped in favor of the systemd ``OnUnitInactiveSec`` and ``ConditionACPower`` options.
```

Comment:
Maybe you could here name the systemd unit name. (I know we have users willing to change a name of the unit, but that's a different problem.)

Reply:
Thank you, added.